restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/Starting Restcomm-Connect.adoc

```diff
@@ -69,7 +90,7 @@ Then you should change the default password.
 
```

Comment:
Please replace MMS with Restcomm Media Server as developers won't know what MMS is